content/docs/reference-react.md

```diff
@@ -13,43 +13,43 @@ redirect_from:
   - "docs/top-level-api-zh-CN.html"
 ---
 
-`React` is the entry point to the React library. If you load React from a `<script>` tag, these top-level APIs are available on the `React` global. If you use ES6 with npm, you can write `import React from 'react'`. If you use ES5 with npm, you can write `var React = require('react')`.
+`React` merupakan _entry point_ untuk menggunakan _library_ React. Apabila anda mengimpor React menggunakan _tag_ `<script>`, _API_ tingkat atas ini tersedia pada `React` _global_. Apabila anda menggunakan _ES6_ dengan _npm_, anda dapat mengimpor React dengan menuliskan `import React from 'react'`. Apabila anda menggunakan _ES5_ dengan _npm_, anda dapat mengimpor React dengan menuliskan `var React = require('react')`.
 
-## Overview {#overview}
+## Ikhtisar {#overview}
 
-### Components {#components}
+### Komponen {#components}
 
-React components let you split the UI into independent, reusable pieces, and think about each piece in isolation. React components can be defined by subclassing `React.Component` or `React.PureComponent`.
+Komponen React membuat anda dapat memisah antarmuka pengguna menjadi independen, dapat digunakan kembali, dan dapat dipikirkan setiap bagiannya secara terpisah. Komponen React dapat didefinisikan dengan melakukan _subclassing_ pada kelas  `React.Component` atau `React.PureComponent`.
```

Comment:
I think `independent, reusable pieces` is a connected phrase. Having a difficulty to understand this part in Indonesian: `Memisah antarmuka pengguna menjadi independen...` 

Masukan:
```Komponen React membuat Anda dapat membagi antarmuka pengguna menjadi sebuah bagian yang independen dan dapat dipergunakan kembali, serta...```

Still can't find a good word for `and think about each piece in isolation` :(